galsim/deprecated/__init__.py

```diff
@@ -15,3 +15,29 @@
 #    this list of conditions, and the disclaimer given in the documentation
 #    and/or other materials provided with the distribution.
 #
+
+# Note: By default python2.7 ignores DeprecationWarnings.  Apparently they are really
+#       for python system deprecations.  I think GalSim deprecations are more appropriately
+#       considered UserWarnings, which are not ignored by default.
+class GalSimDeprecationWarning(UserWarning):
+    def __init__(self, s):
+        super(GalSimDeprecationWarning, self).__init__(self, s)
+
+def depr(f, v, s1, s2=None):
+    import warnings
+    s = str(f)+' has been deprecated since GalSim version '+str(v)+'.'
+    if s1:
```

Comment:
Oh I see, you use an empty string for functionality that has no replacement.  Never mind.